app/routers/credits.py

```diff
@@ -0,0 +1,32 @@
+from fastapi import APIRouter, Request
+import json
+from typing import List
+
+
+from loguru import logger
+from starlette.templating import _TemplateResponse
+
+from app.dependencies import templates
+
+router = APIRouter()
+
+
+def data_from_json() -> List:
```

Comment:
`credits_from_json` - Usually using the word `data` in variable is a red flag :)

Reply:
fixed :)